src/components/pages/search/PlatformFilter/PlatformFilterBadge.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react";
+import { Badge } from "@adjust/components";
+
+import type { PlatformFilterBadgeProps, PlatformLabelValue } from "./types";
+
+import "./platform-filter.css";
+
+const PlatformFilterBadge: FC<PlatformFilterBadgeProps> = ({ items }) => {
+  const currentChoice = items[0];
+
+  const getBadgeColor = (value: PlatformLabelValue) => {
+    if (value === "all") {
+      return "neutral";
+    }
+
+    return `primary`;
+  };
+
+  return (
+    <div
+      className="font-body"
+      style={{ display: "flex", alignItems: "center" }}
+    >
+      <span className="text-base font-medium">Platform</span>
+      <div className="search_platform-filter_badge text-sm">
```

Comment:
This is another place where we could probably style this inline, just to cut down on arbitrary CSS.

```suggestion
      <div className="text-sm [&_span]:font-body [&_span]:text-sm">
```

Reply:
Fixed